scripts/mina-local-network/send-graphql-query.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python
```

Comment:
Please don't produce additional files without the strong need.
This scrip only sends `HTTP POST` requires which can be easily done with the help of `cURL` command (extracted into another function in main script file with that additional output if needed).

Reply:
I created this because if I don't know how to do embed the query into a json directly on bash. This would require some quote escape magic because the query itself has a lot of double quote in it. I thought about using curl at first but the quote escape is a problem here.

Reply:
I created this because if I don't know how to do embed the query into a json directly on bash. This would require some quote escape magic because the query itself has a lot of double quote in it. I thought about using curl at first but the quote escape is a problem here.

Reply:
Got it, thanks. Maybe will be good to refactor it.